client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -217,7 +217,7 @@ const BrewRenderer = (props)=>{
 					onScroll={getCurrentPage}
 					onKeyDown={handleControlKeys}
 					tabIndex={-1}
-					style={{ height: state.height }}>
+					style={{ height: "100vh" }}>
```

Comment:
If a js variable isn't needed here, does the height need to be set in the jsx file at all?  Why not just set the `height` property in the brewRenderer.less file?

Reply:
The height seems to be used all throughout the component, for different methods, including scroll functions,  i am not familiar with the internal workings of this file, and i preferred not to "fix" something that wasn't broken.

I changed this instance because a hardcoded value in pixels was affecting the layout greatly.

Reply:
removed height state, unnecessary, you were right